data/styles/clear/include/Icons.mapcss

```diff
@@ -645,13 +655,15 @@ area|z15-[amenity=water_point]
 
 /* 3. TRANSPORT */
 
-node|z12-[railway=halt],
-node|z14-[railway=station],
-node|z16-[railway=subway_entrance]
+area|z11-[building=train_station],
+node|z12-[railway=station],
```

Comment:
Will have a mess with texts here. And it conflicts with current subways. Let's make here more conservative change to start draw texts:
area|z12-[building=train_station],
node|z13-[railway=station],

Reply:
Or:
node|z12-[railway=station][!transport]

Reply:
I had to add `node|z14-[railway=station][transport=subway]` also, otherwise subway labels disappeared on all zooms